src/api/tokensApi.js

```diff
@@ -8,28 +8,86 @@
 import requestExplorerServiceV1 from './axiosInstance';
 import { get } from 'lodash';
 
+const handleResponse = (response) => {
+  // If status is not retrieved, we assume an internal error ocurred, giving the status code 500
+  // Currently 200 is always returned for success responses
+  if (get(response, 'status', 500) !== 200) {
```

Comment:
Should we compare using `> 299` instead of `!== 200`, to make it future-proof? (in case someone changes the API response codes)

Reply:
That's a great idea, refactored in 2009107edd0fd40fa0d0d94a662f13706632d90c